source/docs/installation.md

```diff
@@ -79,5 +75,21 @@ At the time of writing this guide, the output looked like this:
 
 <pre><code class="shell">3.2.1</code></pre>
 
+## Installing Embark @next
+
+While running `npm install -g embark` will give us the latest and great stable version of Embark, we're sometimes interested in trying out features that are in active development. The Embark maintains a `next` distribution tag on npm that can be used to install versions of Embark that aren't stable.
+
+<pre><code class="shell">$ npm -g install embark@next</code></pre>
+
+## Installing Embark's latest nightly
+
+If you're interested in getting whatever has landed last in the code base, we can go ahead and install the latest version of Embark's GitHub repository like this:
+
+<pre><code class="shell">$ npm -g install github.com/embark-framework/embark</code></pre>
+
+{% note warn Warning: %}
+Keep in mind that both, latest nightlies as well as the current `@next` are considered unstable and may have some bugs. Please install those at your own risk.
+{% endnote %}
+
 Awesome! We're all set up. Let's build our first decentralized application!
```

Comment:
General thought/observation:

the style might be better overall if you / you're *-OR-* we / we're are used consistently, vs. switching back and forth between them.